### PR TITLE
Add winrate trend

### LINKED ISSUE #427: Player win rate trend graph
This is a feature request. 
The graph will help to figure out if a player is actually winning/losing a lot of games lately.

### PR DESCRIPTION
**Pull request by masad-frost**
fixes #427

### CODE REVIEW

File: src/actions/player/playerTrendsActions.js [hunk 1]
```diff
@@ -33,33 +36,60 @@ export const getPlayerTrendsError = (payload, id) => ({
   id,
 });
 
+const getCumalativeSums = (matches, fieldName) =>
```
Comment (howardchung) [inline src/actions/player/playerTrendsActions.js]: Cumulative

File: src/actions/player/playerTrendsActions.js [hunk 2]
```diff
@@ -33,33 +36,60 @@ export const getPlayerTrendsError = (payload, id) => ({
...
+    }
+
+    return cumulativeList;
+  }, []);
+
 export const getPlayerTrends = (playerId, options = {}, fieldName) => (dispatch) => {
   dispatch(getPlayerTrendsRequest(playerId));
   return fetch(`${API_HOST}${getUrl(playerId, options, url)}`)
     .then(response => response.json())
     .then((response) => {
```
Comment (howardchung) [inline src/actions/player/playerTrendsActions.js]: can you just .map() the response to add a field called win_rate with a value of 0 or 1?  Then you can just reuse the existing logic?

File: src/actions/player/playerTrendsActions.js [hunk 3]
```diff
@@ -33,33 +36,60 @@ export const getPlayerTrendsError = (payload, id) => ({
...
+        : getCumalativeSums(matches, fieldName);
 
       return cumulativeSums.map((value, index) => {
         const match = matches[index];
         return {
           x: index + 1,
-          value: Number((value / (index + 1)).toFixed(2)),
-          independent_value: match[fieldName],
+          value: winRateTrend
+            ? (value * 100).toFixed(2)
```
Comment (howardchung) [inline src/actions/player/playerTrendsActions.js]: I think this formatting should be done at the display level.  You can just keep the raw data here?

File: src/components/Player/Pages/Trends/Trends.jsx [hunk 4]
```diff
@@ -16,64 +16,76 @@ import heroes from 'dotaconstants/build/heroes.json';
...
+        >
+          <TrendGraph
+            columns={columns}
+            name={selectedTrend}
+            tooltip={{
+              contents: (d) => {
+                const data = columns[d[0].index];
+                return `<div class="${styles.tooltipWrapper}">
+                <div class="${styles.value}">
+                  ${selectedTrend === 'win_rate' ? '' : strings.trends_tooltip_average}
```
Comment (howardchung) [inline src/components/Player/Pages/Trends/Trends.jsx]: do the number formatting here

File: src/actions/player/playerTrendsActions.js (hunk 2 above)
Comment (masad-frost) [inline src/actions/player/playerTrendsActions.js]: I was fixated on this equation, which coming to think of it doesn't even make sense. We can do the division later

**Pull request merged**